apps/aehttp/test/aehttp_client.erl

```diff
@@ -14,22 +14,22 @@ request(OpId, Params, Cfg) ->
     BaseUrl = string:trim(proplists:get_value(Interface, Cfg), trailing, "/"),
     Path = endpoints:path(Method, OpId, Params),
     Query = endpoints:query(Method, OpId, Params),
-    inets:start(httpc, [{profile, test_browser}]),
-    Response = request(Method, BaseUrl, Path, Query, Params, [], [{timeout, 15000}], [], Cfg),
-    inets:stop(httpc, test_browser),
+    {ok, ClientPid} = inets:start(httpc, [{profile, test_browser}], stand_alone),
```

Comment:
could you now also leave out {profile, test_browser} altogether? Not needed, but you don't seem to need it any more.

Reply:
> could you now also leave out {profile, test_browser} altogether?

[Not possible](https://github.com/erlang/otp/blob/OTP-20.2.3/lib/inets/src/http_client/httpc.erl#L456-L459).

And it makes sense not to use the default profile, as otherwise [standalone process would attempt to re-create (hence failing) the table of keepalive sessions](https://github.com/erlang/otp/blob/OTP-20.2.3/lib/inets/src/http_client/httpc_manager.erl#L421) of the default profile.

----

I performing a few unrelated fine-tuning.